_config/session-manager.yml

```diff
@@ -0,0 +1,8 @@
+---
+Name: auditor-session-manager
+Only:
+  moduleexists: silverstripe/session-manager
+---
+SilverStripe\SessionManager\Control\LoginSessionController:
+  extensions:
+    - SilverStripe\Auditor\AuditHookSessionManager
```

Comment:
Could this be moved to https://github.com/silverstripe/silverstripe-session-manager/pull/34/files along with the extension?
Might make more sense to keep it there and only load it when the auditor module is installed.

Either way, not too fussed..

Reply:
Normally I'd say yes, however auditor is a bit of a weird module in that it has the modules extensions within the auditor modules instead of what's seems like the more sensible way around.

The auditor readme says `Auditor module installs a series of extension hooks into the Framework to monitor activity of authenticated users`

Auditor currently has extension hooks with 'only module exists' checks within yml:
- silverstripe/cms
- silverstripe/mfa

So for for the sake of consistency I think the extension hook should be in auditor, rather than session-manager